### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by WalterSobchak9515** [Weatherlights/Winget-AutoUpdate-Intune#37] [General]
Is this project dead?  I was looking to implement this but don't want to go through the trouble of setting up the intune version if it is no longer being maintained and updated from the original project.

It appears that the project was being updated very regularly up until October 2023 and hasn't been updated since when the original project has received many, many updates since October 2023.

Anyone have any information on this?

**Comment by Karl-WE** [comment]
I was too in the situation to compare and there are honestly too many good 3p tools and helpers for Intune.

Noticed that the same that this fork had unique feature now being implemented into the parent project.
Which also had stale times, and this marked the possible reason for a fork instead of PRs on main.

Would be glad to learn how and where this project still differentiates @Weatherlights .

No shame here if there are reasons to not keeping it up. Hope you are doing good. 

Thanks a ton for the your massive amount of work and good reputation through the Intune (MVP) community and Deskmodder.

**Comment by Weatherlights** [comment]
No it is not but I had some struggle with Microsoft submission policies lately mainly since a new code signing certificate was used and smartscreen decided not to like it... However I am currently working on it again.

The main differences to the original project is:
- Easy deployment through the Microsoft Store (Including updates)
- Plugins fully available through the ADMX template and do not require external storage
- All code is signed (including Mods)

**Reply by Karl-WE** [reply]
Thanks!

**Reply by WalterSobchak9515** [reply]
Ok, great thanks!

I do have a question regarding deployment through the store.

We can't add the store app to deploy via intune as we don't have the option for "Microsoft Store app (new)".  We only have the option in intune to use "Microsoft Store app (legacy)".  I assume this is because we use gcch and this "new" is only available in the commercial version of intune.

So when using intune to try to create an app using the microsoft store, this is what we get:

![image](https://github.com/Weatherlights/Winget-AutoUpdate-Intune/assets/164514930/a05a6179-4722-47cb-b896-f374ddd033a3)

Any ideas on how we can use intune and the app in the store to deploy to clients automatically?  From what I can see it appears I will have to create an app from the installer msi file and deploy it that way.

**Reply by ne-tlw** [reply]
Please use the provided msi-file. This works well even with all admx policies.

**Reply by WalterSobchak9515** [reply]
Ok, when do you think there will be an updated MSI?  The current one that is there is a bit outdated.

**Reply by Weatherlights** [reply]
It is still stable so you can use it without any problems. The new version can easily be updated by simply uploading the new MSI version to the LoB app.

However the new version is currently in certification with Microsoft. The preview version is already certified and tested and the new version will hopefully become available next week. As mentioned: I replaced my Code Signing certificate and since my name has an ö in it some certificate authorities issue me a certificate with Gotze and some with Götze... this minor change resulted in the fact that my new code signature has not inherited the reputation my old one had. So it took me a little more time to get my app through the Smartscreen Whitlisting process and the Microsoft Store.

I am positive that the new release will make the Code Signature well known so all future releases will work without a dedicated whitelisting by Smartscreen.

**Reply by Weatherlights** [reply]
And finally... after two more rounds with Microsoft the certification went through. Luckly new releases may not cause that much trouble since I am now an absolute expert in getting stuff through Smartscreen ;).

**Reply by WalterSobchak9515** [reply]
Great, thank you very much!

**Reply by Weatherlights** [reply]
Very welcome. There is now also a preview release so you can validate upcoming changes aswell.

**Reply by hotzenwalder** [reply]
I very much like this project and the work you put into it. Since WAU was updates to version 2.0, do you expect to use this version in your fork?

**Comment by WalterSobchak9515** [comment]
I do have another question that is kind of unrelated to this thread.

As I mentioned before above, I can't use the MS store app (even though I would like to, MS doesn't make that possible).  When I run WAU, it tells me in the logs that "WAU AutoUpdate" is disabled:

`09:04:45 - WAU AutoUpdate is Disabled.`

I was looking through the intune policies and I don't see an option for allowing WAU to auto update via intune policies using your ADMX template .

Is there a way within intune to set this to all machines so that WAU auto updates itself when it needs to?? (I don't have the option to use the store app for deploying to mass PC's, I know you have stated previously somewhere on here that it will auto update if you use the store app).

Or perhaps there is a mods configuration that I can set that will allow it to autoupdate itself?

**Reply by WalterSobchak9515** [reply]
Any ideas on how to make sure that WGAU stays up to date with this intune msi version since I can't use the store app?

**Reply by Weatherlights** [reply]
The easiest approach would be to use an MSI LoB App.. You can simply Upgrade WAUC by editing your app and upload the next MSI file and then just safe. The MSI supports inplace upgrades and does not require any special caution. Of course this is still somewhat manual but it should take you only 3-5 minutes to just edit, upload msi and click finish :).

**Reply by WalterSobchak9515** [reply]
Ok, thanks.  I was looking to see if there was a way to make it auto update.  I will just publish a new msi each time.

For the notifications, I have them completely disabled, but folks are still getting things like this in their windows notifications area:

![image](https://github.com/user-attachments/assets/e93e33bc-2e79-48ce-ae00-0e66ed62de10)

Why is it still throwing out notifications when they have been totally disabled in the intune settings?

![image](https://github.com/user-attachments/assets/3fb6082e-661a-4100-b9aa-ed5be126969f)

**Reply by WalterSobchak9515** [reply]
Just got this as well:

![image](https://github.com/user-attachments/assets/cda973c0-367e-4f85-9d5a-60a5a9212d94)

Firefox updated too and it showed notification for starting and finishing the update.
